sdk/storage/azdatalake/internal/generated/constants.go

```diff
@@ -6,4 +6,4 @@
 
 package generated
 
-const ServiceVersion = "2024-05-04"
+const ServiceVersion = "2024-08-04"
```

Comment:
Auto-generated code needs to be updated for having this version in the `x-ms-version` header

Reply:
generated Datalake code